### PR TITLE
Feature/loading modal

### PR DESCRIPTION
**Pull request by PorlyBe**
- Show loading modal
- Tweak some components

image

image

### CODE REVIEW

File: src/components/Icons/Spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react";
+import styled, { keyframes } from "styled-components";
+
+import spinner from "../../assets/spinner.png";
+
+const portalSpin = keyframes`
```
Comment (micahpatng) [inline src/components/Icons/Spinner.tsx:6]: should be just `spin` `loading` or whatever, I think?

File: src/components/Label.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
+
+export const Label = styled.span<{ $secondary?: boolean }>`
```
Comment (micahpatng) [inline src/components/Label.tsx:3]: i'm genuinely curious why we use `$` here?

File: src/components/Icons/Spinner.tsx (hunk 1 above)
Comment (PorlyBe) [reply src/components/Icons/Spinner.tsx:6]: loool.. copy pasta fail

File: src/components/Label.tsx (hunk 2 above)
Comment (PorlyBe) [reply src/components/Label.tsx:3]: it's the latest recommended way to do props with styled-components... so they dont conflict with DOM... see here https://styled-components.com/docs/basics#passed-props

**Review by micahpatng — APPROVED** [review]

**Pull request merged**